src/leap/mail/mail.py

```diff
@@ -190,11 +212,15 @@ def get_subpart(self, part):
 
         sub_pmap = self._pmap.get("part_map", {})
         try:
-            part_map = sub_pmap[str(part + 1)]
+            part_map = sub_pmap[str(part)]
         except KeyError:
-            logger.debug("getSubpart for %s: KeyError" % (part,))
+            log.msg("getSubpart for %s: KeyError" % (part,))
```

Comment:
yes, this has been something quick to get the logbook solution capturing them.
I think in the near future we should log specifically from each module (soledad/keymanager/mail) to a file or file-like object, and probably have the centralized logging facility integrating all (something like having tabs, or the ability to see all of them together in the same timeline could be nice)

@ivanalejandro0 some food for thought ^^ in case you want to rescue this for some issue